_posts/2021-02-15-fix-intermittently-failing-ci-builds-flaky-tests-rspec.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Fix intermittently failing CI builds due to flaky tests in RSpec"
+date:   2021-02-15 12:00:00 +0100
```

Comment:
Note the hour. If you merge PR before 12:00 on Monday then the article won't be visible.

The article won't be automatically published after 12:00 hours because Jekyll is static HTML compiled during deployment. You would have to commit a new change to force deployment and a new compilation of the blog content to make this blog post visible.

I recently started using hour like 8:00 so when I merge on Monday then it's already published.